server/neptune/workflows/internal/rebase/workflow.go

```diff
@@ -0,0 +1,46 @@
+package rebase
+
+import (
+	"time"
+
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/github"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/terraform"
+	workflowMetrics "github.com/runatlantis/atlantis/server/neptune/workflows/internal/metrics"
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	RebaseGithubRetryCount    = 3
+	RebaseStartToCloseTimeout = 10 * time.Second
+)
+
+type Request struct {
+	Repo github.Repo
+	Root terraform.Root
+}
+
+type rebaseActivities struct {
+	*activities.RevsionSetter
+	*activities.Github
+}
+
+func Workflow(ctx workflow.Context, request Request) error {
+	var r *rebaseActivities
+	scope := workflowMetrics.NewScope(ctx, "rebase")
```

Comment:
You don't need this:
https://github.com/lyft/atlantis/blob/32a8c4d91b293c0a4250a28acafaa9d0aeab3fb9/server/neptune/temporal/worker.go#L145-L146